src/components/TsmlUI.tsx

```diff
@@ -64,174 +63,169 @@ export default function TsmlUI({
   });
 
   const { settings, strings } = mergeSettings(userSettings);
+  const [searchParams, _] = useSearchParams();
 
-  // enable forward & back buttons
-  useEffect(() => {
-    const popstateListener = () => {
-      setState({ ...state, input: getQueryString(settings) });
-    };
-    window.addEventListener('popstate', popstateListener);
+  //   // update canonical
+  //   let canonical = document.querySelector('link[rel="canonical"]');
+  //   if (!canonical) {
+  //     canonical = document.createElement('link');
+  //     canonical.setAttribute('rel', 'canonical');
+  //     document.getElementsByTagName('head')[0]?.appendChild(canonical);
+  //   }
+  //   canonical.setAttribute(
+  //     'href',
+  //     formatUrl(
+  //       state.input.meeting ? { meeting: state.input.meeting } : state.input,
+  //       settings
+  //     )
+  //   );
```

Comment:
it's fine to leave this here for now. curious though if there's something that needs to be figured out for this?